packages/sanity/src/components/previews/portableText/BlockPreview.tsx

```diff
@@ -65,7 +64,7 @@ export function BlockPreview(props: PreviewProps<'block'>) {
             </Box>
           )}
 
-          {actions}
+          {actions as React.ReactNode}
```

Comment:
I dunno, I think these casts need a comment or something, cause at face value this seems like it could hide runtime errors?

Reply:
As mentioned in another comment I'll change this to any as it needs a follow up fix with a proper typings fix later. For now we just need to be able to land the react v18 PR and stop further scope creep 😬